environment

```diff
@@ -77,6 +77,7 @@ EVENT_RELAY_AWS_USERNAME="dss-event-relay-${DSS_DEPLOYMENT_STAGE}"
 EVENT_RELAY_AWS_ACCESS_KEY_SECRETS_NAME="event-relay-user-aws-access-key"
 GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME="gcp-credentials.json"
 GOOGLE_APPLICATION_SECRETS_SECRETS_NAME="application_secrets.json"
+DSS_TERRAFORM_BACKEND_BUCKET_TEMPLATE="org-humancellatlas-dss-config-{account_id}"
```

Comment:
Can you add a one-liner comment above explaining when and with what `{account_id}` is interpolated and that it's OK to remove it, too. It took me a while to figure this out. And I had the context in mind.

Reply:
Added comment.